alpine/aarch64/Dockerfile

```
@@ -14,7 +14,7 @@ ENV \
 SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 
 # Version
-ARG BASHIO_VERSION=0.11.0
+ARG BASHIO_VERSION=0.12.0
```

Now available with the `/tmp/.bashio` tweak

```suggestion
ARG BASHIO_VERSION=0.12.1
```